libs/velo-external-db-core/src/converters/aggregation_transformer.ts

```diff
@@ -2,11 +2,16 @@ import { AdapterAggregation, AdapterFunctions } from '@wix-velo/velo-external-db
 import { IFilterTransformer } from './filter_transformer'
 import { projectionFunctionFor } from './utils'
 import { errors } from '@wix-velo/velo-external-db-commons'
-import { Aggregation, Group } from '../spi-model/data_source'
+import { Operation, Aggregation, Filter } from '../spi-model/data_source'
```

Comment:
I'll wait with the review to aggregation transformer until the aggregation_transformer tests would change as well in order to understand the changes better.